r2i/tensorflow/engine.cc

```diff
@@ -217,7 +217,21 @@ std::shared_ptr<r2i::IPrediction> Engine::Predict (std::shared_ptr<r2i::IFrame>
   }
 
   std::shared_ptr<TF_Tensor> pout_tensor (out_tensor, TF_DeleteTensor);
-  prediction->SetTensor (pgraph, out_operation, pout_tensor);
+
+  //TODO: here should implement an interation logic to add all the possible outputs
```

Comment:
Why this was kept as TODO?

Reply:
I removed this TODO comment, I set that comment at that time to make clear that there some implementation that had to be done in the soon future.